src/components/Home.jsx

```diff
@@ -1,11 +1,58 @@
-import React from 'react';
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import './Home.scss';
+import TrendingArticles from './presentation/TrendingArticles';
+import LatestArticles from './presentation/LatestArticles';
+import { getArticles } from '../actions/articles';
+import PreLoader from './presentation/PreLoader';
+import FeaturedArticles from './presentation/FeaturedArticles';
 
-const Home = () => (
-  <div className="App">
-    <header className="App-header">
-      <p>{'Welcome to Author\'s Haven by Team Balder!'}</p>
-    </header>
-  </div>
-);
+class Home extends Component {
+  state = {
+    loading: true,
+  }
 
-export default Home;
+  async componentDidMount() {
+    const { getArticles: fetchArticles } = this.props;
+    await fetchArticles();
+    this.setState({ loading: false });
+  }
+
+  render() {
+    const { articles } = this.props;
+    const { loading } = this.state;
+    const trending = articles.slice(0, 5);
+    const featured = articles.slice(5, 7);
+    const latest = articles.slice(7, 10);
+
```

Comment:
Is this the best way to logically implement 25 - 27?

Reply:
You didn't answer my question.

Reply:
Hi @omokaro-faith, sorry for the late response. Yes, this is not the best approach, but like I mentioned last week, we currently do not have trending/featured articles.